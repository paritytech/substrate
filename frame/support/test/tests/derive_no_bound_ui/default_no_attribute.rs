trait Config {
	type C;
}

#[derive(frame_support::DefaultNoBound)]
enum Foo<T: Config> {
	Bar(T::C),
	Baz,
}

fn main() {}
