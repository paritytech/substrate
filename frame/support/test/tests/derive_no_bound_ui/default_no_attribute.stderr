error: no default declared, make a variant default by placing `#[default]` above it
 --> tests/derive_no_bound_ui/default_no_attribute.rs:6:6
  |
6 | enum Foo<T: Config> {
  |      ^^^
